runhouse/servers/http/http_server.py

```diff
@@ -461,6 +462,32 @@ async def get_call(
                 serialization=serialization,
             )
 
+    # `/logs` POST endpoint that takes in request and LogParams
+    @staticmethod
+    @app.get("/logs/{run_name}/{serialization}")
```

Comment:
what serialization options are relevant for logs?

Reply:
in the event of exception we still need to serialize with the expected type